web/yaamp/core/functions/yaamp.php

```diff
@@ -251,6 +253,7 @@ function getAlgoPort($algo)
 		'm7m'		=> 6033,
 		'veltor'	=> 5034,
 		'velvet'	=> 6133,
+		'velvet'	=> 3233,
```

Comment:
error here

Reply:
haha oops forgot to change it.

Reply:
Sorry about that, I use a custom yiimp so I can't just directly push any changes I make. I'll definitely re-review my code in the future.